src/Swagger.ObjectModel/SwaggerRoot.cs

```diff
@@ -29,8 +29,8 @@ public class SwaggerRoot : SwaggerModel
         /// <remarks>
         /// The value MUST be an existing Swagger specification version.
         /// </remarks>
-        [SwaggerProperty("swagger", true)]
-        public string SwaggerVersion
+        [SwaggerProperty("openapi", true)]
+        public string OpenAPI
```

Comment:
OpenAPIVersion?

Reply:
I guess we can name it to be more clear - I was just going with the same name the specification used, but just changed the capitalization.